source/glkernel/include/glkernel/sample.h

```diff
@@ -21,6 +21,8 @@ size_t poisson_square(tkernel<glm::tvec2<T, P>> & kernel, unsigned int num_probe
 template <typename T, glm::precision P>
 size_t poisson_square(tkernel<glm::tvec2<T, P>> & kernel, T min_dist, unsigned int num_probes = 32);
 
+template <typename T, glm::precision P>
```

Comment:
in addition, please describe the algorithm, e.g.:
* pixel area is subdivided into n x n strata
* one sample per stratum
* stratified (jittered) sampling reduces clustering
of samples, non-uniform sampling of areas and
missing of small details

taken from http://cg.informatik.uni-freiburg.de/course_notes/graphics2_04_sampling.pdf